.github/workflows/release.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           fetch-depth: 0
       - uses: olafurpg/setup-scala@v10
       - uses: olafurpg/setup-gpg@v3
-      - run: sbt ci-release
+      - run: sbt test ci-release
```

Comment:
Do we want both the test workflow and the release workflow? Would we just have the release workflow depend on the test workflow?

Reply:
This helps to mitigate a commit directly to master from being released if the tests fail

Ie  - do we/you ever want to release to a package/version if there's failing tests?

Reply:
Well thats why I wonder if we can have the test workflow happen before the release workflow, and only have the test workflow for master branch. Im new to github actions so I dont know

Reply:
With a PR the "test" workflow would happen first.
With a direct merge to master, the "test" happens before the "release".

This ensure the "release" only happens with the passing of "test' - ie, a transient CI (actions) error could cause a failure in release, but more than likely it would catch a transient CI issue/flakey test.